src/multiselect.js

```diff
@@ -91,6 +95,8 @@ export class Multiselect {
       Shortcut.unregisterShortcut();
       Shortcut.registerOurShortcut(this.useCopyPasteCrossTab_);
     }
+    // Register workspace comments
+    Blockly.ContextMenuItems.registerCommentOptions();
```

Comment:
We shouldn't register comments in our plugin anyway, it should be called by the developer manually, so remove these